=== modified file 'src/signond/signond.conf'
Index: signon-8.48/src/signond/signond.conf
===================================================================
--- signon-8.48.orig/src/signond/signond.conf	2013-03-19 10:47:28.639570090 -0400
+++ signon-8.48/src/signond/signond.conf	2013-03-19 10:47:28.635570090 -0400
@@ -20,6 +20,7 @@
 FileSystemName=signonfs
 Size=8
 FileSystemType=ext2
+SecretsStorage=gnome-keyring
 
 [ObjectTimeouts]
 ; All the values are in seconds
